src/ViewSpecialCases.elm

```diff
@@ -46,8 +48,14 @@ specialCases =
         , ( "archives"
           , \model ->
                 (div []
-                    [ p [] [ text archivesIntro ]
-                    , ViewHelpers.renderArchives
+                    [ span [] [ text archivesIntro ]
+                    , input
+                        [ Html.Events.onInput UpdateField
+                        , Html.Attributes.placeholder "Search"
+                        , Html.Attributes.class "searchInput"
```

Comment:
For HTML events & attributes I do like to expose them.

So the imports should be:

```elm
import Html.Attributes exposing (class, href, placeholder)
import Html.Events exposing (onInput)
```

And then just use `onInput`, `placeholder` and `class` in the code.